spec/requests/rack_attack_spec.rb

```diff
@@ -98,59 +105,156 @@
           expect(request.env['REMOTE_ADDR']).to eq('1.2.3.4')
         end
 
-        it 'suppresses the subnet' do
+        it 'suppresses the subnet with custom response' do
           key = "rack::attack:#{(Time.now.to_i/@period).to_i}:req/subnet:1.2.3."
           Rack::Attack.cache.store.increment(key, 999)
           @epoch_time = Time.now.to_i
 
           get '/about', {}, 'REMOTE_ADDR' => '1.2.3.5'
           data = { :count => 1001, :limit => 1000,
                    :period => @period.to_i, :epoch_time => @epoch_time }
-          expect(response.status).to eq(429)
-          expect(response.headers['Retry-After']).to eq(@period.to_s)
```

Comment:
minor: we don't want to keep that header?

Reply:
Either way is fine. Just making sure it wasn't an oversight. Since we return 503 now, we can omit it.